modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -301,7 +301,7 @@ class NDB_Form_User_Accounts extends NDB_Form
                 //modify examiner radiologist to be in the binary format 0-1
                 $ex_radiologist = $ex_radiologist === 'Y' ? 1:0;
 
-                $examinerID = $DB->pselect(
+                $examinerID = $DB->pselectOne(
```

Comment:
this is just a clean up, noticed that I can be even more specific than pselectCol since only one actual value can be returned